blocks/cards/cards.css

```diff
@@ -1,27 +1,137 @@
 .cards > ul {
-  list-style: none;
-  margin: 0;
-  padding: 0;
-  display: grid;
-  grid-template-columns: repeat(auto-fill, minmax(257px, 1fr));
-  grid-gap: 24px;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    flex-flow: column wrap;
+    justify-content: space-between;
 }
 
-/* .cards > ul > li {
-  border: 1px solid #dadada;
-  background-color: var(--background-color);
-} */
-
-.cards .cards-card-body {
-  margin: 16px;
+.cards > ul > li {
+    border-left: none;
+    border-top: var(--qui-alias-border-width-thin) solid var(--qui-alias-color-border-default);
+    box-sizing: border-box;
+    display: block;
+    max-width: 100%;
 }
 
-.cards .cards-card-image {
-  line-height: 0;
+.union-palette .cards > ul > li,
+.cards.union-palette  > ul > li {
+    border-top-color: var(--qui-color-union-blue-400);
+    padding: var(--qui-spacing-16) var(--qui-spacing-0) var(--qui-spacing-12) var(--qui-spacing-0) !important;
 }
 
 .cards > ul > li img {
-  width: 100%;
-  aspect-ratio: 4 / 3;
-  object-fit: cover;
+    transform: scale(2.7);
+    box-sizing: border-box;
+    margin-left: var(--qui-spacing-20);
+    margin-bottom: var(--qui-spacing-40);
+}
+
+.cards .card-image {
+    padding-top: var(--qui-spacing-40) !important;
+    padding-bottom: var(--qui-spacing-8) !important;
+    padding-left: var(--qui-spacing-0);
+}
+
+.cards .card-container {
+    display: flex;
+    flex-direction: column;
+    flex-grow: 1;
+}
+
+.cards .card-title {
+    padding-bottom: var(--qui-spacing-16);
+    margin-bottom: var(--qui-spacing-0);;
+}
+
+.cards .card-container:first-child {
+    padding-top: var(--qui-spacing-24) !important;
+}
+
+.cards .card-text {
+    margin-bottom: var(--qui-spacing-0);;
+}
+
+.cards .card-button {
+    padding-top: var(--qui-spacing-16);
+    padding-bottom: var(--qui-spacing-16);
+}
+
+.union-palette .card-button{
+    padding-bottom: var(--qui-spacing-0);
+}
+
+@media screen and (width >= 960px) {
+    .cards > ul {
+        flex-flow: row nowrap;
+        justify-content: flex-start;
+        gap: var(--qui-spacing-40);
+    }
+
+    .cards > ul li {
+        display: flex;
+        flex-direction: column;
+        max-width: 400px;
```

Comment:
This doesn't look correct.
The cards have different widths now on smaller screens.
I think this should be either a percentage, or using their grid system classes.